index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Nduati Kagiri</title>
+    <link rel="stylesheet" href="style.css">
+  </head>
+  <body>
+    <nav>
```

Comment:
- [ ] Nice job with the header, kindly add the logo and nav in a header tag for better use of semantics.